tfhe/Cargo.toml

```diff
@@ -116,6 +116,9 @@ x86_64-unix = ["x86_64", "seeder_unix"]
 aarch64 = ["generator_aarch64_aes"]
 aarch64-unix = ["aarch64", "seeder_unix"]
 
+# Example-specific features
+sha256_bool_ladner_fischer = []
+
```

Comment:
let's avoid adding a feature, please add clap as a dev dependency and update the example to parse a flag from the command line to chose one carry propagation version over the other, thanks

Reply:
Done!

Reply:
thanks !